src/tests/stub_CachePeer.cc

```diff
@@ -13,5 +13,7 @@
 
 #include "CachePeer.h"
 
+void CachePeer::rename(const char *) STUB
```

Comment:
Please remove the empty line above this to match the stubs implementation style.

Reply:
Done in 076f60f.